.ci/daily-config.yaml

```diff
@@ -24,9 +24,6 @@ build:
         ${{ env.MAVEN_COMMAND }}
         ${{ env.WORKSPACE }}/clean-up.sh `pwd`
   - project: kiegroup/process-migration-service
-    skip: true
-  - project: kiegroup/optaweb-vehicle-routing
```

Comment:
is it correct to remove these?

Reply:
the same for next one

Reply:
they are still present on project dependecies [1] for `7.67.x.` branch, so if that is correct to remove them we should also remove them from project-dependencies as well. wdyt?

[1] https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/7.67.x/.ci/project-dependencies.yaml#L174

Reply:
yes, i think so, both optaweb's on 7.x are deprecated in products

Reply:
> they are still present on project dependecies [1] for `7.67.x.` branch, so if that is correct to remove them we should also remove them from project-dependencies as well. wdyt?
> 
> [1] https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/7.67.x/.ci/project-dependencies.yaml#L174

I guess we need to remove both from the projects dependencies, it can be done additionally to this PR